server/routes/api.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const router = express.Router()
+const request = require('request')
+const City = require('../model/City')
+const moment = require('moment')
+const { json } = require('body-parser')
+require('dotenv').config()
+
+
+router.get('/city/:cityName',function(req,res){
+    const city=req.params.cityName
+   request(`http://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${process.env.WEATHER_API_KEY}`,
+function (err,response){
+    let data=JSON.parse(response.body)
+    if(data.cod===200){
+        data={
+            name:data.name,
+            temperature:Math.round(data.main.temp),
+            condition:data.weather[0].description,
+            conditionPic:data.weather[0].icon,
+            cod:data.cod
+        }
+        res.send(data)
+    }else{
+        res.send(data)
+    }
+})
+   })
+
+
+
+router.get('/cities',function(req,res){
+    City.find({},function(err,cities){
```

Comment:
use async await here since you are using it in the other routs. Keep thing consistent